eng/pipelines/common/variables.yml

```diff
@@ -9,22 +9,25 @@ variables:
   value: 5.0.201
 - name: LocBranchPrefix
   value: 'loc-hb'
-- name: LocBranch
-  value: 'loc'
 - name: isMainBranch
   value: $[eq(variables['Build.SourceBranch'], 'refs/heads/main')]
 - name: isLocBranch
-  value: $[eq(variables['Build.SourceBranch'], 'refs/heads/loc')]
+  value: $[eq(variables['Build.SourceBranch'], 'refs/heads/loc' or startsWith(variables['Build.SourceBranch'], 'refs/heads/loc-')]
```

Comment:
An error occurred while loading the YAML build pipeline. Unexpected symbol: 'or'. Located at position 54 within expression: eq(variables['Build.SourceBranch'], 'refs/heads/loc' or startsWith(variables['Build.SourceBranch'], 'refs/heads/loc-'). For more help, refer to https://go.microsoft.com/fwlink/?linkid=842996